employee-rostering-frontend/pom.xml

```diff
@@ -147,18 +148,19 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-assembly-plugin</artifactId>
-        <configuration>
-          <descriptors>
-            <descriptor>assembly/sources.xml</descriptor>
-          </descriptors>
-          <appendAssemblyId>true</appendAssemblyId>
-        </configuration>
         <executions>
           <execution>
+            <id>package-sources</id>
```

Comment:
`package-sources`? Isn't it packaging the build output?

Reply:
Yes; probably should rename it `package-build` (was copied from distribution)